cg/models/report/metadata.py

```diff
@@ -0,0 +1,48 @@
+from typing import Optional, Union
+
+from pydantic import BaseModel, validator
+from cg.models.report.validators import validate_empty_field, validate_float
+
+
+class SampleMetadataModel(BaseModel):
+    """
+    Metrics and trending data model associated to a specific sample
+
+    Attributes:
+        million_read_pairs: number of million read pairs obtained; source: StatusDB/sample/reads (/2*10^6)
+        duplicates: fraction of mapped sequence that is marked as duplicate; source: pipeline workflow
+
+    """
+
+    million_read_pairs: Union[None, float, str]
```

Comment:
I'm a bit confused about those Unions, seems like they allow the data to be validated very loosely.. Would it make sense to override the model with a loose one instead when the user wants to force the report?

Reply:
I'm not sure about this, because it will lead to a lot of code duplication, as each model will need to have a complementary one for validation (without the `--force-report` flag). But it's true that pydantic models in the current state only have a formatting role and not so much a validation one.